application/models/QuestionType.php

```diff
@@ -0,0 +1,538 @@
+<?php
+
+/**
+ * Class QuestionType
+ * @property Question $question
+ * @property string $fieldType The type of field question needs for storing data
+ * @property string $fieldDataType numeric vs string types
+ * @property boolean $isText Whether the type is text (string longer than char)
+ * @property boolean $isChar Whether the type is char (one-character-string)
+ * @property boolean $isString Whether the type is string (text or char)
+ * @property boolean $isNumeric Whether the type numeric (integer, double)
+ * @property boolean $isInteger Whether the type integer
+ *
+ * {@inheritdoc}
+ */
+class QuestionType extends StaticModel
```

Comment:
Instead of class QuestionType, shouldn't we have one separate model for each question type?